hledger-lib/Hledger/Reports/PostingsReport.hs

```diff
@@ -92,7 +92,8 @@ postingsReport rspec@ReportSpec{rsOpts=ropts@ReportOpts{..}} j = items
             reportPeriodOrJournalLastDay rspec j
 
       -- Posting report does not use prices after valuation, so remove them.
```

Comment:
The change seems to invalidate this comment - remove/change it ?

Reply:
Will do.